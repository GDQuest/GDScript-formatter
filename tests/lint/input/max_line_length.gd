#region Good

var short_and_sweet = "hello"

#endregion

#region Bad

var really_long_variable_name = "This is a really long string that exceeds the line length of 100 that is configured by default"


func bad():
	var array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24]

	for i in array:
		for j in array:
			for k in array:
				for l in array:
					for m in array:
						print("Here is another very long line, this one deeply nested inside of ton of loops")
#endregion
