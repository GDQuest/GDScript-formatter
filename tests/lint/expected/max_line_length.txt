tests/lint/input/max_line_length.gd:9:max-line-length:warning: Line is too long. Found 128 characters, maximum allowed is 100
tests/lint/input/max_line_length.gd:13:max-line-length:warning: Line is too long. Found 103 characters, maximum allowed is 100
tests/lint/input/max_line_length.gd:20:max-line-length:warning: Line is too long. Found 110 characters, maximum allowed is 100